src/patternfly/components/DataToolbar/data-toolbar-group-toggle-group.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-toolbar__group pf-m-toggle-group{{#if data-toolbar-group--modifier}} {{data-toolbar-group--modifier}}{{/if}}{{#if data-toolbar-group-toggle-group--Reveal}} pf-m-reveal{{data-toolbar-group-toggle-group--Reveal}}{{/if}}{{#if data-toolbar-toggle-group--modifier}} {{data-toolbar-toggle-group--modifier}}{{/if}}"
```

Comment:
Should you use something like `{{#> data-toolbar-group data-toolbar-group--modifier=(concat 'pf-m-toggle-group ' data-toolbar-toggle-group--modifier)}}`

And do you need `{{#if data-toolbar-group--modifier}} {{data-toolbar-group--modifier}}{{/if}}` here? Or could modifiers to this partial be covered by `data-toolbar-toggle-group--modifier`?